.github/workflows/action.yml

```diff
@@ -0,0 +1,17 @@
+name: Check Markdown links
```

Comment:
If we haven't been doing this previously, we might want to temporarily disable it as we have a lot of Markdown files in the core repo that might not have been checked. Cleaning them up is worthwhile, but tedious (ask me how I know).